docs/advanced-topics/metrics.rst

```diff
@@ -105,53 +153,210 @@ The curl command results in something like:
 
 .. code:: bash
 
-   curl -X 'POST' \
+   curl -X 'GET' \
+      'https://dockstore.org/api/api/ga4gh/v2/extended/%23workflow%2Fgithub.com%2Fdockstore%2Fdockstore-tool-bamstats%2Fwdl/versions/1.25-9/execution?platform=TERRA&executionId=2c8c7c45-d4e6-4a0c-891d-a28e7c995c70' \
+      -H 'accept: application/json' \
+      -H 'Authorization: Bearer imafakebearertoken'
+
+If the request was successful, you should receive a ``200`` status code and the execution that you requested, like below:
+
+.. code:: bash
+
+   {
+      "aggregatedExecutions": [],
+      "runExecutions": [
+         {
+            "additionalProperties": null,
+            "cost": null,
+            "cpuRequirements": null,
+            "dateExecuted": "2023-03-31T15:06:49.888745366Z",
+            "executionId": "2c8c7c45-d4e6-4a0c-891d-a28e7c995c70",
+            "executionStatus": "SUCCESSFUL",
+            "executionTime": "PT30S",
+            "memoryRequirementsGB": null,
+            "region": null
+         }
+      ],
+      "taskExecutions": [],
+      "validationExecutions": []
+   }
+
+.. _Update submitted execution:
+
+How do I update workflow metrics?
+---------------------------------
+
+You may want to update metrics that you have previously submitted because you received new metrics for the execution at a later time.
+
+Go to https://dockstore.org/api/static/swagger-ui/index.html#/extendedGA4GH/ExecutionMetricsUpdate and provide your Dockstore token using the lock icon at the top right of the endpoint. This is the endpoint used to update metrics that were submitted Dockstore. Click the "Try it out” button.
+
+First, fill out the parameters. This is where you'll input information about which workflow and version was executed and what platform it was executed on. 
+
+For the ``id``, ``version_id``, and ``platform`` parameters, provide the values that you previously used when submitting the metrics you want to update to Dockstore. This ensures that the correct metrics are updated. For example, to update the metrics that were submitted in the :ref:`Submitting metrics example<Submitting metrics example>`, provide the same ``id``, ``version_id`` and ``platform``.
+
+For the ``description``, you may provide an optional description about the metrics you're updating.
+
+.. figure:: /assets/images/docs/metrics/update-metrics-parameters.png
+    :alt: Query parameters for submitting metrics
+
+In the request body, provide the updated workflow execution metrics that you want to update in Dockstore. Click on Schema to view the schema of the request body. It is the same schema used for :ref:`submitting metrics<Submit metrics request body schema>`.
+
+.. important::
+   Ensure that the execution you are updating have the same execution ID as the execution you previously submitted.
+
+You must provide the full execution object when updating the execution. See :ref:`how to retrieve a submitted execution<View submitted execution>` if you do not have the full execution object.
```

Comment:
That's sort of annoying, but that's REST PUT for you... We've never clean solved this in the web service. Probably using PATCH is the way to go (no need to change anything, this just got me thinking).